Samples/iOS-Swift/iOS-Swift/SentrySDKWrapper.swift

```diff
@@ -242,11 +242,13 @@ extension SentrySDKWrapper {
         
         let fm = FileManager.default
         let dir = "\(appSupportDirectory)/io.sentry/feedback"
-        do {
-            try fm.createDirectory(atPath: dir, withIntermediateDirectories: true)
-        } catch {
-            print("[iOS-Swift] Couldn't create directory structure for user feedback form hook marker files: \(error).")
-            return
+        if !fm.fileExists(atPath: dir) {
```

Comment:
`l`: We should maybe also include the `isDirectory` parameter to check the file exists and is a valid directory